src/enact/__init__.py

```diff
@@ -63,6 +63,7 @@
 from enact.pretty_print import invocation_summary
 
 from enact.references import commit
+from enact.references import commit_async
```

Comment:
Should the other _async methods be exposed?

e.g `checkout_async` etc.

Reply:
Done